src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.i2d.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        internal delegate int GetI2DSizeFunc<in THandle>(THandle handle);
```

Comment:
Instead of I2D (which I believe is "Internal (representation) to DER", howabout Export, or Encode?